Documentation/compilation.rst

```diff
@@ -29,23 +28,23 @@ Add the following lines to your .bashrc (vi ~/.bashrc).
   export LIBRARY_PATH=$HOME/lib:$LIBRARY_PATH
   export LD_LIBRARY_PATH=$HOME/lib:$LD_LIBRARY_PATH
   export PKG_CONFIG_PATH=$HOME/lib/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$HOME
   export EDITOR=vi
   export CPATH=$HOME/include:$CPATH 
 
   # run "exec bash" or "source ~/.bashrc" to apply environment to the current shell
 
-Installing SCons
+Installing CMake
 ----------------
 
 .. code-block:: bash
 
-  wget http://prdownloads.sourceforge.net/scons/scons-3.0.5.tar.gz
-  tar -xaf scons-3.0.5.tar.gz
-  cd scons-3.0.5
-  python setup.py install --prefix=$HOME
-  cd ..
+  wget -qO- https://github.com/Kitware/CMake/releases/download/v3.16.4/cmake-3.16.4-Linux-x86_64.tar.gz | tar -xvz -C "/" && mv "/cmake-3.16.4-Linux-x86_64" "${HOME}/bin/cmake"
```

Comment:
This uses cmake 3.16 - but documentation says 3.15 is needed. Maybe we should decide on one of both for docs/travis